Fedora/input/xccdf/system/accounts/physical.xml

```diff
@@ -122,6 +122,51 @@ the grub2 superuser account and password, please refer to
 <ref nist="IA-2(1),IA-5(e),AC-3" disa="213" cis="1.5.3" />
 </Rule>
 
+<Rule id="bootloader_uefi_password" severity="medium">
+<title>Set the UEFI Boot Loader Password</title>
+<description>The UEFI grub2 boot loader should have a superuser account and password
+protection enabled to protect boot-time settings.
+<br /><br />
+To do so, select a superuser account and password and add them into the
+appropriate grub2 configuration file(s) under <tt>/etc/grub.d</tt>.
```

Comment:
Would it be possible instead of "and add them into the appropriate grub2 configuration file(s) under ..."
to explicitly mention ```/etc/grub.d/01_users``` file? (from https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/System_Administrators_Guide/sec-GRUB_2_Password_Protection.html )

Quoting more for further details:
```
All menu entries can be password-protected against changes by setting superusers,
which can be done in the /etc/grub.d/00_header or the /etc/grub.d/01_users file.
The 00_header file is very complicated and, if possible, avoid making modifications
in this file. Menu entries should be placed in the /etc/grub.d/40_custom and users
in the /etc/grub.d/01_users file. The 01_users file is generated by the installation
application anaconda when a GRUB boot loader password is used in a kickstart
template (but it should be created and used it if it does not exist). Examples in
this section adopt this policy. 
```

From the above looks to me, though it should be in generall not that much important in which specific
file the superusers are defined, by ensuring it will be done in ```/etc/grub.d/01_users_file``` we will
be consistent what the official documentation / recommendation says.